### PR TITLE
Doc Update for Public IP Address

### PR DESCRIPTION
**Pull request by ritika-msft**

### CODE REVIEW

File: 07-Customizing-AzSK-for-your-Org/Readme.md [hunk 1]
```diff
@@ -1555,6 +1555,50 @@ v) Override ControlSettings.json in ServerConfigMetaData.json as shown below:
 
 vi) Rerun the policy update or setup command (the same command you ran for the first-time setup).
 
+### How can we treat each public IP address as an individual resource?
+   Org policy admins can enable public IP address as an individual resource by making following org policy settings:
+
+i) Copy the ControlSettings.json from the AzSK installation to your org-policy folder.
+
+ii) Update the value to true in below tag under PublicIpAddress:
```
Comment (ganesh-msft) [inline 07-Customizing-AzSK-for-your-Org/Readme.md:1563]: Change to - Update the EnablePublicIpResource setting to true  - and in the snip below show it as true

Comment (ritika-msft) [reply 07-Customizing-AzSK-for-your-Org/Readme.md]: Made the required changes

File: 07-Customizing-AzSK-for-your-Org/Readme.md [hunk 2]
```diff
@@ -1555,6 +1555,50 @@ v) Override ControlSettings.json in ServerConfigMetaData.json as shown below:
...
+ii) Update the EnablePublicIpResource setting to true.
+```JSON
+"PublicIpAddress": {
+    "EnablePublicIpResource": true
+   },
+```
+
+iii) Save the file.
+
+iv)  Edit the ServerConfigMetadata.json file in your local org-policy folder and create an entry for "ControlSettings.json" as below:
```
Comment (abhaydaga) [inline 07-Customizing-AzSK-for-your-Org/Readme.md:1572]: what if admins have overridden the control settings.

Comment (ritika-msft) [reply 07-Customizing-AzSK-for-your-Org/Readme.md:1579]: Added the instructions for already overridden ControlSettings.json in step no.1a and step no.5.

**Review by ganesh-msft — APPROVED** [review]

**Pull request merged**